Apps/phawsguardduty/awsguardduty.json

```diff
@@ -170,20 +172,20 @@
                     "order": 0
                 },
                 "finding_id": {
-                    "description": "The ID of the finding",
+                    "description": "The IDs of the findings(Comma separated IDs allowed)",
```

Comment:
Add space between findings and parenthesis.

Reply:
Added